src/globus_cli/commands/transfer.py

```diff
@@ -187,43 +194,47 @@
         "local user account to map to. Only usable with Globus Connect Server v5 "
         "mapped collections."
     ),
+    default=globus_sdk.MISSING,
+    type=OMITTABLE_STRING,
 )
-@click.option("--perf-cc", type=int, hidden=True)
-@click.option("--perf-p", type=int, hidden=True)
-@click.option("--perf-pp", type=int, hidden=True)
-@click.option("--perf-udt", is_flag=True, default=None, hidden=True)
+@click.option("--perf-cc", default=globus_sdk.MISSING, type=OMITTABLE_INT, hidden=True)
```

Comment:
If I'm understanding correctly, the new `omittable` category of types does not do anything special with the omitted case (which is odd based on the name). Rather it requires that developers properly set the default to MISSING in order to function whenever used. Feels like a decently large degradation of contract given the force multiplicative effect.